.github/workflows/ci.yml

```
@@ -31,9 +31,11 @@ jobs:
       - run: npm run test
       - run: npm run e2e-test
         env: 
-          REFRESH_TOKEN: ${{ secrets.REFRESH_TOKEN }}
-          CLIENT_ID: ${{ secrets.CLIENT_ID }}
-          CLIENT_SECRET: ${{ secrets.CLIENT_SECRET }}
+          E2E_TEST_REFRESH_TOKEN: ${{ secrets.E2E_TEST_REFRESH_TOKEN }}
+          E2E_TEST_CLIENT_ID: ${{ secrets.E2E_TEST_CLIENT_ID }}
+          E2E_TEST_CLIENT_SECRET: ${{ secrets.E2E_TEST_CLIENT_SECRET }}
+          E2E_TEST_IDP_URL: ${{ secrets.E2E_TEST_IDP_URL }}
```

We shouldn't be using IdP as a name anywhere anymore (as per my comment in the Inrupt Engineering channel from Dec 18th (I don't want to paste a direct link in here!)). I know we'll have to update the GitHub secret name too, i.e.:
```suggestion
          E2E_TEST_SOLID_IDENTITY_PROVIDER_URL: ${{ secrets.E2E_TEST_SOLID_IDENTITY_PROVIDER_URL }}
```

---

Feel free to make the change, but it was a bit too much effort for me after having already spent quite a bit of time on this - keeping the names the same made it easier to re-use the solid-client credentials. If you do go over these names, could you do the same for solid-client?

---

Ok - I'll approve this PR then, and just create a new ticket to sweep through the codebases and update the names.